tests/unit/s2n_client_hello_test.c

```diff
@@ -91,6 +91,45 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Test s2n_client_hello_has_extension */
+    {
+        {
```

Comment:
Why the double brackets?

Reply:
It is just enables to add multiple tests for the same API. I'm going to remove it.